src/components/insights/drawers/DrawerConfigContentCard.vue

```diff
@@ -139,6 +149,35 @@ export default {
 
       return true;
     },
+    flowResultsOptions() {
+      const selectedFlowUuid = this.config.flow?.[0]?.value;
+
+      if (selectedFlowUuid) {
+        const selectedFlowMedatada = JSON.parse(
+          this.flows.find((flow) => flow.uuid === selectedFlowUuid).metadata,
+        );
```

Comment:
It would be good to use the util for json parse which prevents it from breaking the code if the parameter is not a valid string